util/filter_reflections.py

```diff
@@ -112,6 +112,8 @@ def filter_reflection_table(reflection_table, intensity_choice, *args, **kwargs)
         reducer = AllSumPrfScaleIntensityReducer
     elif all(i in intensity_choice for i in ["sum", "profile"]):
         reducer = SumAndPrfIntensityReducer
+    elif all(i in intensity_choice for i in ["sum | profile"]):
+        reducer = SumORPrfIntensityReducer
```

Comment:
Would this be clearer?
```suggestion
    elif {"sum | profile"} <= set(intensity_choice):
        reducer = SumORPrfIntensityReducer
```

also does intensity_choice need to be a list or can it be a genuine set given that as far as I can tell we don't care about the order of elements?

Reply:
True, this can be a set rather than a list